RealEskalate/Internship-2023
# feat(dartsmith.bereket):create weatherApp

- create presentation, domain and data layer for the weather app.
- test the repository and search usecase
- implement bloc 
- create dependency injection
-  check the network connection
![newimage](https://github.com/RealEskalate/Internship-2023/assets/91548062/66e2f9c7-33f8-47c9-899b-b4062675e229)
![im2](https://github.com/RealEskalate/Internship-2023/assets/91548062/f35cdc34-11c6-4df4-9ace-9a7b652e684f)
